blivet/flags.py

```diff
@@ -40,7 +39,13 @@ def __init__(self):
         #
         # enable/disable functionality
         #
-        self.selinux = selinux.is_selinux_enabled()
+        try:
+            import selinux
+        except ImportError:
+            self.selinux = False
```

Comment:
We might want to log this for situations where SELinux is enabled but `python3-libselinux` is not installed. Thoughts?